developer-docs/bolt-server.md

```diff
@@ -174,61 +174,61 @@ For example:
 If the task runs the response will have status 200.
 The response will be a standard bolt Result JSON object.
 
-## Install from Source
+## Running from source
 
-The following example walks through the setup to get the bolt-server running on an Ubuntu 16.04 vmpooler node.
+From your checkout of bolt run
 
-1. Download and install puppet6 and puppetserver (for certs). The latest release can be found at http://nightlies.puppet.com/
 ```
-curl -O http://nightlies.puppet.com/apt/puppet6-nightly-release-xenial.deb
-dpkg -i ./puppet6-nightly-release-xenial.deb
-apt-get update
-apt-get install puppetserver
-export PATH=$PATH:/opt/puppetlabs/bin
-puppet resource service puppetserver ensure=running
+BOLT_SERVER_CONF=config/local.conf bundle exec puma -C puppet_config.rb
 ```
-2. Get certs (Error messages OK here, as long as certs are generated)
+
+
+setup your environment for running commands with
+
 ```
-puppet agent -t
-puppet cert list -a
-puppet cert sign [host reported from command above]
+export BOLT_CACERT=spec/fixtures/ssl/ca.pem
+export BOLT_CERT=spec/fixtures/ssl/cert.pem
+export BOLT_KEY=spec/fixtures/ssl/key.pem
+export BOLT_ROOT=https://localhost:62658
 ```
-Check that the following certs have been generated
+
+you can now make a curl request to bolt which should have an empty response
+
 ```
-/etc/puppetlabs/puppet/ssl/certs/$HOSTNAME.pem
-/etc/puppetlabs/puppet/ssl/private_keys/$HOSTNAME.pem
+curl -v --cacert $BOLT_CACERT --cert $BOLT_CERT --key $BOLT_KEY $BOLT_ROOT
 ```
-3. Download and install bolt-server. The latest release can be found at http://builds.delivery.puppetlabs.net/pe-bolt-server
+
+## Running in a container
```

Comment:
We should document where the certs are mounted in the container, in case people want to provide their own.

Reply:
I don't think that should block merging this in. It's solely intended for a handful of dev/test usecases. I don't think anything I'd document now would be stable over the next month so it's best to leave it up to the users to come up with new usecases and improve the bolt-server code, container, and docs as needed.